### ISSUE TITLE
Multiplicity of relationships

### ISSUE THREAD

**Issue by RobWalker867** [SEMICeu/Core-Person-Vocabulary#23]
I do not believe that model of person is correct. It implies that many relationships are single-valued, whereas in practice, they can be multiple:  
i)	It assumes that a person has one name (string). However a person can have multiple legitimate names, e.g. a married women might use her maiden name for professional purposes, her married name for legal purposes, and her former name for historic purposes (e.g. relating to children). Most of the attributes of Person in the current model are actually those of Name, and one Person may have many Names. Splitting Person into Person and Name would also enable a history of names to be recorded (important for criminal and other records).  
ii)	It assumes that a person has a single registered address. This is also not correct. Many people have more than one address, often in different jurisdictions. 
iii)	It assumes that a person has a single citizenship. Many people have joint citizenship.

**Comment by makxdekkers** [comment]
Core Vocabularies do not say anything about cardinalities. Any of the properties may be used zero, one or multiple times. Any restrictions to their use can be defined in an application profile for a specific use context. So for a particular application it may be stated that a person can have multiple names, addresses and citizenships. Probably, properties like date and place of birth will by nature only occur a maximum of once, but most of the others may well have many occurrences.

**Comment by RobWalker867** [comment]
The point is that multiple occurrences of properties are not ordered. Therefore there is no connection for example, between a particular family name and a particular given name when more than one of each occur. That is why person and name need to be separate classes, with one person having many names, and each name having a single family name, given name etc.

**Comment by makxdekkers** [comment]
@RobWalker867 Good point. Indeed, the Core Person vocabulary does not support the use cases that you bring up here. You need to be aware that the use case for which the Core Person vocabulary was developed is a simple one; basically to model the information that is captured for example in a passport, a visa, a residence permit, and is used by the [eIDAS Regulation](http://data.europa.eu/eli/reg/2014/910/oj). In those cases, people are registered with one name, one current address and other data that are in essence a snapshot for a particular context and at a particular point in time. Other use cases are of course conceivable but currently not supported by the Core Person vocabulary. I hope this answers your question.

**Comment by dimi-schepers** [comment]
During the Core Vocs webinar dd. 2021-04-23, it was mentioned that the current approach is also not ideal for countries with double family names, because their order cannot be specified.

**Comment by EmidioStani** [comment]
During the [webinar of 21/02/2023](https://joinup.ec.europa.eu/collection/semic-support-centre/event/webinar-core-vocabularies-core-business-core-person-vocabulary-cccev], the issue was discussed in proposing the addition of middle name; however no consensus was reached.
